content/pages/airport-cases.md

```diff
@@ -0,0 +1,41 @@
+Title: The Airport Cases
+Date: 2017-01-31 **:**
+Author: anseljh
+Slug: airport-cases
```

Comment:
Assuming we'll want to create another curated list of cases in the future, we should probably namespace the slug. maybe do: `featured-cases/airport-cases/` and move this file into a `featured-cases` directory (or something similar that sounds better)?